shared-libs/search/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "search",
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
It looks like you've bumped the version but not changed any code?

Reply:
Sorry that happened because I'm about to, but haven't done it as part of this change yet.